types/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import './vue';
+import Vue from 'vue';
+
+export function install (vue: typeof Vue): void
+
+export interface Tour {
+  start(startStep?: string): void
+  previousStep(): void
+  nextStep(): void
+  stop(): void
+  skip(): void
+  finish(): void
+  currentStep: number
```

Comment:
Would be nice to also include the computed property `isRunning`. E.g. I use it to ensure my tours do not overlap.

Reply:
I've added that to the PR now. Please test by putting 
`"vue-tour": "git://github.com/glorat/vue-tour.git#typings",`

in the dependencies section of your package.json and let me know if that works for you